types/list-stream/list-stream-tests.ts

```diff
@@ -0,0 +1,54 @@
+import * as ListStream from "list-stream";
```

Comment:
See previous comment.

Reply:
Should be resolved by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/30669#discussion_r238110651.